### PR TITLE
[Pkg] Add note to description of `com.google.android.packageinstaller`

### LINKED ISSUE #398: Some Huawei devices bootloop after the removal of packageinstaller, add a note
**Your phone**: Huawei Y6 (2019)

**Packages documentation to update:**
```
com.google.android.packageinstaller
```

### com.google.android.packageinstaller
**Current description**
> Occasionally runs in the background.
Related to installing Google-specific packages? Possibly something to do with Play services and backend updates/installations. Installing an app through Google Play worked fine with it disabled.

**Proposed description**
> Occasionally runs in the background.
Related to installing Google-specific packages? Possibly something to do with Play services and backend updates/installations. Installing an app through Google Play worked fine with it disabled.
NOTE: Some Huawei devices bootloop after uninstalling/disabling this package and require a full restore via eRecovery. Be careful!

### PR DESCRIPTION
**Pull request by AnonymousWP**
Fixes #398

### CODE REVIEW

**Comment by 0x192** [comment]
Thanks! 

Ouch! That description was a bit off the mark (my fault). I updated it and moved the package to `Unsafe`.

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -17939,11 +17939,10 @@
   {
     "id": "com.google.android.packageinstaller",
     "list": "Google",
-    "description": "Occasionally runs in the background.\nRelated to installing Google-specific packages? Possibly something to do with Play Services and back-end updates/installations. Installing an app through Google Play worked fine with it disabled.\nNOTE: Some Huawei devices bootloop after uninstalling/disabling this package and require a full restore via eRecovery. Be careful!",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json]: Why did you remove this part if I may ask? It was suggested by the user who created the issue (the "NOTE" part).
